integreat_cms/cms/templates/analytics/_page_hix_widget.html

```diff
@@ -8,32 +8,51 @@
 {% endblock collapsible_box_title %}
 {% block collapsible_box_content %}
     <div class="flex flex-wrap justify-between gap-2">
-        <p>
+        <p id="hix-list-n-pages">
             {% blocktranslate trimmed with n=page_translations|length %}
                 The list shows the first {{ n }} pages with the lowest HIX value in ascending order.
             {% endblocktranslate %}
         </p>
+        <p id="hix-list-all-pages" class="hidden">
+            {% translate "The list shows all pages with the lowest HIX value in ascending order." %}
+        </p>
```

Comment:
```suggestion
            {% blocktranslate trimmed with n=page_translations|length %}
                The list shows the pages with the lowest HIX value in ascending order.
            {% endblocktranslate %}
        </p>
```
Having a table show only a couple of rows until you expand it is such a standard behaviour, I'm not even sure we need this changing textblock in the first place?